pep-9999.rst

```diff
@@ -0,0 +1,1105 @@
+PEP: 9999
+Title: Type Parameter Syntax
+Version: $Revision$
+Last-Modified: $Date$
```

Comment:
We no longer need these headers: https://peps.python.org/pep-0001/#pep-header-preamble

```suggestion
```